demo/rust/filedemo/src/main.rs

```diff
@@ -17,6 +17,7 @@ use std::path::PathBuf;
 
 fn picovoice_demo(
     input_audio_path: PathBuf,
+    access_key: &str,
```

Comment:
can you also expose the `requires_endpoint` parameter to users in demos? I mean users should be able to change it if they want in demos